src/App.jsx

```diff
@@ -23,7 +23,7 @@ function App() {
                     />
                     <Route
                         path={Routes.APARTMENTS_EDIT}
-                        render={() => <ApartmentEdit />}
+                        component={ApartmentEdit}
```

Comment:
I would like to keep this line intact, if you can -- I have some plans about this, i.e. I remember I wrote this code like this for particular reason. If I recall correctly, it was either because of some unnecessary renders invoked when passed as component directly, OR this was kept this way to add Helmet library and handle things like title, breadcrumbs etc. in the feature. Obviously this probably will be changed with migration to the new router version, but let's keept it like it is rn, k? Thanks :)